cmd/bucket-replication.go

```diff
@@ -143,11 +158,22 @@ func hasReplicationRules(ctx context.Context, bucket string, objects []ObjectToD
 	return false
 }
 
+// isStandardHeader returns true if header is a supported header and not a custom header
+func isStandardHeader(headerKey string) bool {
```

Comment:
If the headers are specific to replication then this func should be renamed `isOneOfReplicationHeaders`

Reply:
checking for standard headers here.